flywheel/subsystems/flywheel.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+from commands2 import Command, Subsystem
+from commands2.sysid import SysIdRoutine
+from wpilib import sysid
+
+from phoenix6 import SignalLogger
+from phoenix6.hardware import TalonFX
+from phoenix6.configs import FeedbackConfigs, MotorOutputConfigs
+from phoenix6.controls import VoltageOut
+from phoenix6.signals import NeutralModeValue
+
+from wpimath.units import volts
+
+from constants import TalonIds
+
+
+class Flywheel(Subsystem):
+    FLYWHEEL_GEAR_RATIO = 24.0 / 18.0
+
+    def __init__(self) -> None:
+        self.flywheel = TalonFX(TalonIds.shooter_flywheel)
```

Comment:
Crazy idea: we dependency inject the motor controller and gear ratio, so we can use this [for the swerve steer too](https://github.com/thedropbears/pysysid/pull/11#issuecomment-1962171554)